### PR TITLE
webui: tests: add method to partition disk

### PR DESCRIPTION
**Pull request by tomasmatus**

### CODE REVIEW

File: ui/webui/test/check-storage [hunk 1]
```diff
@@ -408,12 +419,8 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         disk = "/dev/vda"
         dev = "vda"
         btrfsname = "btrfstest"
+        self.partition_disk(disk, [["--new=0:0:+1MiB", "--typecode=0:ef02"], "--new=0:0:+1GiB", "--new=0:0:+10GiB", "--new=0:0:0"])
```
Comment (KKoukiou) [inline ui/webui/test/check-storage:422]: I would prefer to have it a bit more user friendly with the partitiion disk method taking parameters the partition sizes and the format types. So something like:
`partition_disk(disk,  [("biosboot", "1MiB"), ("ext4", "1GB"), ("btrfs", "10GB"), ("xfs", "")])`

Empty last size means use the rest. What do you think? I think that's more readable.

Comment (tomasmatus) [reply ui/webui/test/check-storage:422]: Ok!, I'll look into it

File: ui/webui/test/check-storage [hunk 2]
```diff
@@ -408,17 +433,7 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
...
-        sgdisk --new=0:0:+1MiB -t 0:ef02 {disk}
-        sgdisk --new=0:0:+1GiB {disk}
-        sgdisk --new=0:0:+10GiB {disk}
-        sgdisk --new=0:0:0 {disk}
-        mkfs.ext4 {disk}2
-        mkfs.xfs -f {disk}3
-        mkfs.btrfs -f -L {btrfsname} {disk}4
-        """)
-
+        self.partition_disk(disk,  [("1MiB", "biosboot"), ("1GB", "ext4"), ("10GiB", "xfs", "-f"), ("", "btrfs", "-f", "-L", btrfsname)])
```
Comment (KKoukiou) [inline ui/webui/test/check-storage:436]: You can safely always use the -f/-F flags. The inconsistency in the tests is not delibarate. take care `-f` is for btrfs and xfs. ext4 has `-F`

Comment (KKoukiou) [reply ui/webui/test/check-storage:436]: So just to be clear - don't make the extra mkfs flags parameter - this is not needed. Always force the mkfs.

Comment (tomasmatus) [reply ui/webui/test/check-storage]: done

**Review by KKoukiou — APPROVED** [review]

File: ui/webui/test/check-storage [hunk 3]
```diff
@@ -395,6 +395,37 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         udevadm settle --timeout=120
         """)
 
+    def partition_disk(self, disk, partitions_params):
```
Comment (VladimirSlavik) [inline ui/webui/test/check-storage:398]: It might make sense to describe what is expected in `partition_params`.

**Review by VladimirSlavik — APPROVED** [review]
Thank you! LGTM.

File: ui/webui/test/check-storage [hunk 4]
```diff
@@ -395,6 +395,37 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
...
+
+            sgdisk.append(disk)
+
+            command += f"\n{' '.join(sgdisk)}"
+
+            if params[1] != "biosboot":
+                mkfs = [f"mkfs.{params[1]}"]
+
+                # force flag
+                if params[1] in ["xfs", "btrs"]:
```
Comment (vojtechtrefny) [inline ui/webui/test/check-storage:415]: typo: btr**f**s

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me otherwise.

File: ui/webui/test/check-storage (hunk 4 above)
Comment (tomasmatus) [reply ui/webui/test/check-storage:415]: oh yikes, thanks for catching this

File: ui/webui/test/check-storage (hunk 3 above)
Comment (tomasmatus) [reply ui/webui/test/check-storage:398]: Is simple comment fine or would you prefer a proper docstring?

Comment (VladimirSlavik) [reply ui/webui/test/check-storage:399]: A comment is enough, thank you!

**Pull request merged**